src/ResourceManager/SubscriptionDefinition/Commands.SubscriptionDefinition/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,50 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Commands.SubscriptionDefinition")]
+[assembly: AssemblyDescription("")]
```

Comment:
@wilcobmsft please fill out some of these missing values - you can view the [`AssemblyInfo.cs` file](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Compute/Commands.Compute/Properties/AssemblyInfo.cs#L20-L23) for the Compute project as a reference

Reply:
Done.